cli/deployment/config_test.go

```diff
@@ -125,20 +125,22 @@ func TestConfig(t *testing.T) {
 	}, {
 		Name: "OIDC",
 		Env: map[string]string{
-			"CODER_OIDC_ISSUER_URL":    "https://accounts.google.com",
-			"CODER_OIDC_EMAIL_DOMAIN":  "coder.com",
-			"CODER_OIDC_CLIENT_ID":     "client",
-			"CODER_OIDC_CLIENT_SECRET": "secret",
-			"CODER_OIDC_ALLOW_SIGNUPS": "false",
-			"CODER_OIDC_SCOPES":        "something,here",
+			"CODER_OIDC_ISSUER_URL":            "https://accounts.google.com",
+			"CODER_OIDC_EMAIL_DOMAIN":          "coder.com",
+			"CODER_OIDC_CLIENT_ID":             "client",
+			"CODER_OIDC_CLIENT_SECRET":         "secret",
+			"CODER_OIDC_ALLOW_SIGNUPS":         "false",
+			"CODER_OIDC_SCOPES":                "something,here",
+			"CODER_OIDC_IGNORE_EMAIL_VERIFIED": "true",
 		},
 		Valid: func(config *codersdk.DeploymentConfig) {
 			require.Equal(t, config.OIDC.IssuerURL.Value, "https://accounts.google.com")
 			require.Equal(t, config.OIDC.EmailDomain.Value, "coder.com")
 			require.Equal(t, config.OIDC.ClientID.Value, "client")
 			require.Equal(t, config.OIDC.ClientSecret.Value, "secret")
-			require.Equal(t, config.OIDC.AllowSignups.Value, false)
+			require.False(t, config.OIDC.AllowSignups.Value, false)
```

Comment:
This still has the `, false` although it changed from Equal to False, intentional?

Reply:
Nope, although `require.False` is more idiomatic regardless. Updated.